Formula/t/technitium-dns.rb

```diff
@@ -0,0 +1,68 @@
+class TechnitiumDns < Formula
+  desc "Self host a DNS server for privacy & security"
+  homepage "https://technitium.com/dns/"
+  url "https://github.com/TechnitiumSoftware/DnsServer/archive/refs/tags/v13.5.0.tar.gz"
+  sha256 "7a5f47af4c938c5983e415547b777f853ede5ada8837012c8b5e8e36df1a380e"
+  license "GPL-3.0-or-later"
+
+  depends_on "dotnet@8"
```

Comment:
any issue tracking the latest dotnet support?

Reply:
- https://github.com/TechnitiumSoftware/DnsServer/issues/1303

Reply:
Yeah, I had asked the same question: https://github.com/TechnitiumSoftware/DnsServer/discussions/1294#discussioncomment-12802718